encrypted-media-respec.html

```diff
@@ -4038,14 +4049,14 @@
             Potentially, this could reveal information about a user to another user of the same device, including potentially the <a def-id="origin">origins</a> that have used a particular Key System (i.e. sites visited) or even the content that has been decrypted using a Key System.
           </p>
           <p>If information stored by one origin affects the operation of the Key System for another origin, then potentially the sites visited or content viewed by a user on one site may be revealed to another, potentially malicious, site.</p>
-          <p>If information stored for one browsing profile, browser, or user account on the client device affects the operation of the Key System for other profiles, browsers, or accounts, then potentially the sites visited or content viewed by in one may be revealed by or correlatable with another profile, browser, or account.</p>
+          <p>If information stored for one <a def-id="user-agent-profile"></a> on the client device affects the operation of the Key System for other <a def-id="user-agent-profile">user agent profiles</a>, or browsers, then potentially the sites visited or content viewed in one may be revealed by or correlatable with another <a def-id="user-agent-profile">user agent profiles</a> or browser.</p>
```

Comment:
Although specifically mentioned in the definition of UAP,  I think the [operating system] user account part is especially relevant here and worth explicitly mentioning.

"Browsing profile" also seems particularly clearer here.